src/bindings/python/src/pyopenvino/frontend/frontend.cpp

```diff
@@ -22,15 +22,16 @@ void regclass_frontend_FrontEnd(py::module m) {
 
     fem.def(
         "load",
-        [](FrontEnd& self, const std::string& s) {
-            return self.load(s);
+        [](FrontEnd& self, const py::object& path) {
```

Comment:
Could you also add such feature to other places like `Core.compile_model` where path is used as argument?